projects/ngx-powerbi/tsconfig.lib.json

```diff
@@ -2,27 +2,15 @@
   "extends": "../../tsconfig.json",
   "compilerOptions": {
     "outDir": "../../out-tsc/lib",
-    "target": "es2015",
-    "module": "es2015",
-    "moduleResolution": "node",
     "declaration": true,
-    "sourceMap": true,
     "inlineSources": true,
-    "emitDecoratorMetadata": true,
-    "experimentalDecorators": true,
-    "importHelpers": true,
-    "types": [],
-    "lib": [
-      "dom",
-      "es2015"
```

Comment:
Reason for removal of these lines? Was this done automatically on upgrade to Angular 9.x?

Reply:
I think this was done during the upgrade.

Reply:
I can review them and added them back again if you want.

Reply:
it's ok, as long as it works